libs/common/src/types/mongo-db-models/facility.ts

```diff
@@ -52,6 +52,7 @@ export type Facility = {
   paymentType: string;
   createdAt: UnixTimestamp;
   updatedAt: UnixTimestamp;
+  bondType?: string; // only featured in BOND facilities
```

Comment:
@BethThomas141 @MarRobSoftwire  do we want to start extracting these out into a bond facility type and union the types here??

Reply:
I think we should look to do this at some point but it might be quite a bit of work to separate out all the values. Needs some thought about how to do this as an iterative approach rather than all in one

Reply:
I've started updating all instances of `bondType` to use constants, I could do this at the same time.

I don't think such a PR would be _too_ bad, there are ~90 instances of hard coded `bondType` in ~60 files that'd be replaced, would just be repetitive

Reply:
@AlexBramhill agree we should do this, let's do it as a separate PR!

Reply:
Will resolve this here, but a conversation to pick up in another PR

Reply:
I've created DTFS2-7607 for this 👍